chroma-manager.conf.template

```diff
@@ -159,11 +159,12 @@ server {
     location /influx {
         limit_except GET { deny all; }
         auth_request /auth;
-        proxy_set_header Host $http_host;
-        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_set_header Host $host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-Host $host;
         proxy_set_header X-Forwarded-Server $host;
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
-        proxy_pass http://localhost:8086/query;
+        proxy_pass {{INFLUXDB_PROXY_PASS}}/query;
```

Comment:
Tested these changes work in RPM based setups?

Reply:
Will verify